mods/d2k/rules/structures.yaml

```diff
@@ -800,6 +806,39 @@ palace:
 		DisplayRadarPing: True
 		CameraActor: camera
 		ActivationSequence:
+	ProduceActorPower@fremen:
+		Description: Recruit Fremen
+		LongDesc: Elite sniper infantry unit                       \n  Strong vs Infantry\n  Weak vs Vehicles\n  Special Ability: Invisibility
```

Comment:
> 21:48 (+pchote) oh, I see 
21:48 (+pchote) abcdefg30: remove your hotkey bindings 
21:48 (+pchote) which aren't there by default 
21:48 (+pchote) they are adding the padding for you